src/library/VTKExtensions/Core/vtkF3DConfigure.h.in

```diff
@@ -7,7 +7,6 @@ static const std::string F3D_RESERVED_STRING = "f3d_reserved";
 static const std::string F3D_EXIT_HOTKEY_SYM = "Escape";
 
 #cmakedefine01 F3D_WINDOWS_GUI
```

Comment:
why is `F3D_WINDOWS_GUI` not moved to the application?

Reply:
we need it in the vtk extensions